k8s/k8s.go

```diff
@@ -294,6 +296,45 @@ func CmdAddK8s(args *skel.CmdArgs, conf utils.NetConf, nodename string, calicoCl
 	return result, nil
 }
 
+// ContainerIDMismatchErr is for when ContainerID does not match ActiveInstanceID during Deletion process.
+var ContainerIDMismatchErr = errors.New("ContainerID does not match ActiveInstanceID")
+
+// CmdDelK8s performs some special checks for the "DEL" operation on a kubernetes pod.
+// The following logic only applies to kubernetes since it sends multiple DELs for the same endpoint.
+// We store CNI_CONTAINERID as ActiveInstanceID in WEP Metadata for k8s,
+// so we need to make sure we check if ContainerID and ActiveInstanceID are the same before deleting the pod.
+func CmdDelK8s(c *calicoclient.Client, ep api.WorkloadEndpointMetadata, args *skel.CmdArgs, logger *log.Entry) (*api.WorkloadEndpointMetadata, error) {
+
```

Comment:
I think we've had this before, but I don't like like the first line of a function to be blank.